app/src/main/java/io/github/joshkergan/giftr/db/MappingTableContract.java

```diff
@@ -0,0 +1,26 @@
+package io.github.joshkergan.giftr.db;
+
+import android.provider.BaseColumns;
+import android.provider.ContactsContract;
+
+/**
+ * Created by Patrick on 16/10/2016.
+ * Contract for the Mapping Table, which maps the Item table to the People table
+ */
+
+public final class MappingTableContract{
+    // No external instantiation
+    private MappingTableContract() {
+    }
+
+    /* Inner class to define the table components */
+    public static class MappingTableEntry implements BaseColumns{

```

Comment:
Oh I see. Still think the table bit is redundent, since we reference it using MappingTableContract.MappingEntry. So I think that's fine like that.